src/main/java/org/jenkinsci/plugins/github/util/JobInfoHelpers.java

```diff
@@ -31,10 +34,20 @@ private JobInfoHelpers() {
      *
      * @return predicate with true on apply if job contains trigger of given class
      */
-    public static Predicate<AbstractProject> withTrigger(final Class<? extends Trigger> clazz) {
-        return new Predicate<AbstractProject>() {
-            public boolean apply(AbstractProject job) {
-                return job.getTrigger(clazz) != null;
+    public static Predicate<Job> withTrigger(final Class<? extends Trigger> clazz) {
+        return new Predicate<Job>() {
+            public boolean apply(Job job) {
+                if (job instanceof ParameterizedJobMixIn.ParameterizedJob) {
```

Comment:
See [this document](https://github.com/jenkinsci/workflow-plugin/blob/master/COMPATIBILITY.md#historical-background).